frontend/utils/time.js

```diff
@@ -43,3 +43,11 @@ export function compareISOTimestamps (a: string, b: string): number {
   const B = new Date(b).getTime()
   return A > B ? 1 : (A < B ? -1 : 0)
 }
+
+export function humanDate (
+  datems = Date.now(),
+  opts = { month: 'short', day: 'numeric' }
+) {
+  const locale = navigator.languages ? navigator.languages[0] : navigator.language
```

Comment:
Small: can you move this `const` outside of this function so that the logic is not performed on each invocation of the function?

Reply:
Done. But the locale logic still needs to be done inside the fn at least once, otherwise, the server will break (navigator does not exist there). Question. Will we support SSR on GI?